erdos/op.py

```diff
@@ -86,14 +92,22 @@ def spin(self):
     def log_event(self, processing_time, timestamp, log_message=None):
         pass
 
+    def log_streams(self, stream_uid, msg):
+        self.loggers[stream_uid].info(msg)
+
     def _add_input_streams(self, input_streams):
         """Setups and updates all input streams."""
         self.input_streams = self.input_streams + input_streams
+        if self.log_input:
+            for stream in self.input_streams:
+                self.loggers[stream.uid] = setup_logging(stream.uid, log_file="{}.log".format(stream.uid))
```

Comment:
I think we actually want to `pickle.dump()` to a file rather than use a logger. If we use loggers, we won't be able to replay the streams.